apis/swagger.yml

```diff
@@ -1787,6 +1787,11 @@ definitions:
         type: "boolean"
         x-nullable: false
         default: true
+      DisableHostsfiles:
```

Comment:
`Hostsfiles` may not so good, do we have more proper name for `/etc/hosts`, `/etc/hostname`, `/etc/resolve.conf` files?  😆 

cc @allencloud @rudyfly

Reply:
I do not like the naming of `DisableHostsfiles `, either. How about `DisableLocalFiles` or `DisableNetworkFiles`?

Reply:
DisableNetworkFiles +1

Reply:
agree it